README.md

```diff
@@ -104,7 +128,41 @@ Gtdownload Release 3.8.7
 ICGC Storage Client Version: 1.0.13
 ICGC-Get Version: 0.5
 ```
-## Unit Tests
+# Repository specific configuration
+Most configuration options can either be overwritten through the command line or environmental variables.
+Environmental variables are in all caps, have underscores as separators, and are prefixed by ICGCGET_.
+Command line options have dashes as separators and are prefixed by two dashes.  Config file options 
+have periods as separators.  The only exception to this rule is the icgc api path.  It can only
+be modified through the config file.
+
+The config file can be specified either by passing an absolute path to the config to the 
+command line with `--config`, or by making an environmental variable `ICGCGET_CONFIG` that contains 
+the absolute path.  If neither of these options are chosen, the tool will look for  `.icget/config.yaml`
+in your home directory.
+
+All clients require an absolute path to your local client installation under repo.path in the config file or 
+ICGCGET_REPO_PATH as an environmental variable.  All clients support the ability to configure the number of 
+data streams to use when downloading under repo.transport.parallel or REPO_TRANSPORT_PARALLEL
+Most clients can be made to download using the UDT protocol by using the repo.udt config option.
+### Collaboratory and aws-virigina:
```

Comment:
Can we please add a link to external documentation for each of these? For example, for ICGC it would be http://docs.icgc.org/cloud/guide/. For EGA it might be https://www.ebi.ac.uk/ega/about/access and / or http://docs.icgc.org/portal/access/#raw-sequencing-and-other-primary-data. Etc. Basically links to get the user started with more detailed information.

I'm thinking something that summarizes the compliance office for each (dbGap, ERA Commons, DACO, etc.) would be useful.